contracts/LiquidityBridgeContract.sol

```diff
@@ -409,11 +517,17 @@ contract LiquidityBridgeContract {
             s := mload(add(signature, 0x40))
             v := byte(0, mload(add(signature, 0x60)))
         }
+		// SDL: I'm think we should not use this signature format
+		// but an EIP712 compatible format.
         bytes memory prefix = "\x19Ethereum Signed Message:\n32";
         bytes32 prefixedHash = keccak256(abi.encodePacked(prefix, hash));
         return ecrecover(prefixedHash, v, r, s) == addr;
     }
 
+	// SDL: Sadly this is not the format that I designed for the bridge.
+	// I designed it to be a double hash but it's a single hash.
+	// We may have problems in the future introducing versioning, because
+	// of the variable size.
```

Comment:
Can this be solved by hashing the quote again?

Reply:
Let's discuss this in a call.

Reply:
we need to do this call. I think we already have scheduled it